tests/ffi/selinux/test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../common/prepare.sh
+
+disk_cleanup
+prepare_test
+reload_config
+
+#Preset with expected value the variable.
+expected_value="setenforce:  security_setenforce() failed:  Permission denied"
+
+# Get setenforce 0 permission denied from qm.
+setenforce_0=$(podman exec -it qm setenforce 0)
+
+echo "This is what qm returns $setenforce_0"
+
+#Check if setenforce 0 succeeds in QM container and fail the test if it does.
+if      [[ $setenforce_0 != *"$expected_value"* ]];then
+        echo "Attempt to change Selinux enforcement to 0 succeeded inside QM container. Test failed."
+        exit 1
+fi
+
+echo "Attempt to change Selinux enforcement to 0 denied successfully inside QM container."
```

Comment:
include a clear PASS/FAIL status for each test, something like:
info_message "PASS: your_function_name: ......"

Reply:
Done